src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationService.kt

```diff
@@ -60,8 +69,17 @@ class CalculationService(
     trancheCommencementDate: LocalDate?,
     tranche: SDSEarlyReleaseTranche,
     trancheOneCommencementDate: LocalDate,
+    trancheDates: TrancheOne,
```

Comment:
calles tranche dates, but it's actually the tranche itself?

Reply:
As discussed - will raise a tech debt ticket - and have renamed